issue518_refactoringDatabaseProfiles

FWuellhorst opened the pull request:
- Removed underscores of filenames to match the current Naming guidelines of AixLib
- Updated the docstring of AixLib/DataBase/Profiles/Profile_BaseDataDefinition.mo according to the one proposed in the issue
- made the with-spaces consistent

Closes #518

PMehrfeld commented:
@FWuellhorst : Thanks, this looks much better. Now, only the [latest conversion script](https://github.com/RWTH-EBC/AixLib/blob/issue518_refactoringDatabaseProfiles/AixLib/Resources/Scripts/ConvertAixLib_from_0.5.3_to_0.5.4.mos) must be enhanced by a `convertClass`-statement. Here you find a [Wiki-Link](https://github.com/RWTH-EBC/AixLib/wiki/Conversion-Scripts).

FWuellhorst commented:
@PMehrfeld Thanks for the review. I added the conversion script according to the wiki and updated the package.mo file.
Hope everything is correct now.
